pulpcore/cli/ansible/context.py

```diff
@@ -78,6 +79,8 @@ class PulpAnsibleRepositoryContext(PulpRepositoryContext):
     SYNC_ID = "repositories_ansible_ansible_sync"
     MODIFY_ID = "repositories_ansible_ansible_modify"
     VERSION_CONTEXT = PulpAnsibleRepositoryVersionContext
+    # TODO specify correct version of ansible plugin
+    CAPABILITIES = {"pulpexport": [PluginRequirement("ansible", "999.0.0")]}
```

Comment:
@daviddavis @gerrod3 do you have a proper version statement for this?

Reply:
Looks like it got added in 0.7.0:

https://github.com/pulp/pulp_ansible/blob/main/CHANGES.rst#features